src/databricks/labs/ucx/install.py

```diff
@@ -193,16 +202,22 @@ def run(self):
         workspace_installation.run()
 
     def configure(self) -> WorkspaceConfig:
-        try:
-            return self._installation.load(WorkspaceConfig)
-        except NotFound as err:
-            logger.debug(f"Cannot find previous installation: {err}")
         logger.info("Please answer a couple of questions to configure Unity Catalog migration")
         HiveMetastoreLineageEnabler(self._ws).apply(self._prompts)
+
         inventory_database = self._prompts.question(
             "Inventory Database stored in hive_metastore", default="ucx", valid_regex=r"^\w+$"
         )
 
+        if self._check_inventory_database_exists(inventory_database):
+            raise RuntimeWarning(f"Inventory database with name {inventory_database} already exists")
```

Comment:
This is a regression.

Reply:
Do we not need this or this is a regression because of the other code change (i.e. not returning the installation). If its the latter, should this logic be after the check for existing installation and force_install overrides?